docs/guides/mParticle-migration-guide.md

```diff
@@ -0,0 +1,299 @@
+2. Update SDK implementation to send data to Amplitude
+3. Validate the migration is successful
+
+!!!note "Table of Contents"
+    1. [Add a Source](#add-a-source)
+    2. [Update SDK implementation](#update-sdk-implementation)
+    3. [Validate events](#validate-events)
+    4. [Add a Destination](#add-a-destination)
+    5. [Migration checklist](#migration-checklist)
+    6. [Frequently asked questions](#frequently-asked-questions)
+
+Quickly review the offerings that are available for mParticle and how that stacks up to Amplitude. 
+
+| mParticle                                                                         | Amplitude |
+|-----------------------------------------------------------------------------------| ----------- |
+| [Connections](https://docs.mparticle.com/guides/platform-guide/connections/)      | [Sources](/data/sources/) & [Destinations](/data/destinations/) |
+| [Audiences](https://docs.mparticle.com/guides/platform-guide/audiences/overview/) | [Audiences](https://help.amplitude.com/hc/en-us/sections/360011146031-Amplitude-Audiences) |
+|                                                                              | [Data Management](https://help.amplitude.com/hc/en-us/categories/5078631395227-Amplitude-CDP) |
+
+!!!info "Recommended Best Practice"
+    Follow a strict release process and [configure multiple environments](https://help.amplitude.com/hc/en-us/articles/5078848559259-Configure-and-manage-your-Amplitude-Data-settings#the-environments-tab). Validate changes within each environment before deploying.
+
+## Add a Source
+
+You can add a [new source](/data/sources/) in just a few clicks.
+
+1. From Data, click **Sources** in the Connections section.
+2. Click **Add Source**.
+3. Browse or search for the source you want to add.
+4. Follow the on-screen prompts.
+
+For detailed instructions, see the documentation for the [source](/data/sources/) you want to add.
+
+## Update SDK implementation
+
+Both mParticle and Amplitude SDKs are meant to capture first party data by tracking user interactions. For the most part both work pretty similarly except some nuances around syntax. Here is high level mapping of concepts between mParticle & Amplitude.
+
+| mParticle | Amplitude | Notes                                      |
+|----------|-----------|--------------------------------------------|
+| app_key  | api_key   | Unique key to validate source of the data. |
+| Workspace | Project   | [Projects](https://help.amplitude.com/hc/en-us/articles/360058073772-Create-and-manage-organizations-and-projects#create-a-project) allow you to organize your data.  |
+| User     | User      | User who is performing action.             |
+| Identify | Identify  | [Identify](/analytics/what-is-amplitude/#user-properties-are-details-about-your-user) updates properties/attributes of the user.|
+| logEvent | Event     | [Event](/analytics/apis/http-v2-api-quickstart/) in Amplitude tracks the action user is performing.|
```

Comment:
This is a little confusing to me. I feel it mixes the payload with methods.

I feel `logEvent` => `track`, the others make sense but I think we should be consistent to show methods or objects.

Reply:
method name changes in both mparticle and our sdks based on language. How about just leaving it as Event in both?